src/AbstractDifferentiation.jl

```diff
@@ -30,6 +45,13 @@ primal_value(::AbstractBackend, ys, ::Any, ::Any) = primal_value(ys)
 primal_value(x::Tuple) = map(primal_value, x)
 primal_value(x) = x
 
+"""
+    AD.derivative(ab::AD.AbstractBackend, f, xs::Number...)
+
+Compute the derivatives `ds` of `f` wrt the numbers `xs` using the backend `ab`. 
+
+Return a tuple `ds` of derivatives, one for each element in `xs`.
```

Comment:
The name is not needed and does not add any important information, I think? IMO (matter of taste) it's also a bit strange to use the imperative form twice.
```suggestion
Compute the derivatives of `f` with respect to the numbers `xs` using the backend `ab`.

The function returns a `Tuple` of derivatives, one for each element in `xs`.
```

Reply:
I tried to copy-paste as much as possible from the README, leaving style improvements to future PRs once the documentation is in place (diffs will be easier to analyze). What do you think?

Reply:
But there are no docstrings yet, are there? I thought this is a new addition.

Reply:
Since you already reviewed the style, I might as well commit your changes. Just explaining that my goal was only moving stuff around at first ;)

Reply:
It is a new feature which copypastes exactly what was in the README. My thought process was moving stuff first, improving stuff next. But we're on track to do both so let's forge ahead!